Guide_1_Send_Payment.md

```diff
@@ -1,31 +1,52 @@
 # Guide: Making a Payment
 
-_(Note: Are there any specific steps required for different payment schemes (SWIFT etc.)? Do the steps in this guide work the same for any scheme? Should be mentioned somehow)_
+_**(Note: Are there any specific steps required for different payment schemes (SWIFT etc.)? Do the steps in this guide work the same for any scheme? Should be mentioned somehow)**_
 
 In this guide you will learn how to use the Form3 Payments API to make a payment. The guide covers the following steps:
 - Send the payment
 - Track the payment progress manually
 - Track the payment progress automatically using subscriptions
 
+Each step is illustrated with Python code snippets, so you can execute each step as you read along. The snippets are ready-to-run programs and work in Python's interactive console.
+
 ## Prerequisites:
 Before you start, make sure you have the following things ready to go:
-- API credentials. Contact your Form3 contact to obtain your credentials. _(Note: Better name for this role? Who do they have to contact?)_
+- API credentials. Contact your Form3 contact to obtain your credentials. _**(Note: Better name for this role? Who do they have to contact?)**_
 - Register your bank ID and BIC with Form3
-- Sign up for Form3's scheme simulator. This is a sandbox environment that you can use to simulate transactions in order to test your application. _(Note: in the API this is called "payment simulator". Which term should we use?)_
+- Sign up for Form3's scheme simulator. This is a sandbox environment that you can use to simulate transactions in order to test your application. _**(Note: in the API this is called "payment simulator". Which term should we use?)**_
```

Comment:
"Sign up for Form3's scheme simulator for FPS payments." I think scheme simulator is maybe a better term, as we have different implementations for different schemes.